libnavigation-base/src/main/java/com/mapbox/navigation/base/internal/NavigationConstants.kt

```diff
@@ -0,0 +1,255 @@
+package com.mapbox.navigation.base.internal
+
+import androidx.annotation.IntDef
+import androidx.annotation.StringDef
+
+/**
+ * Navigation constants
+ *
+ * @since 0.1.0
+ */
+object NavigationConstants {
```

Comment:
Are any of these constant used in multiple modules?

Reply:
for now might make it internal 👍